tests/units/classes/Request/ClassesForTests.php

```diff
@@ -0,0 +1,146 @@
+<?php
+
+use Lizmap\Request\OGCRequest;
+use Lizmap\Request\Proxy;
+use Lizmap\Request\WFSRequest;
+use Lizmap\Request\WMSRequest;
+use Lizmap\Request\WMTSRequest;
+
+class ProjectForOGC extends ProjectForTests
+{
+    public $loginFilters;
+
+    public function getRelativeQgisPath()
+    {
+    }
+
+    public function getLoginFilters($layerName, $edition = false)
+    {
+        return $this->loginFilters;
+    }
+
+    public function setData($key, $value)
+    {
+        $this->data[$key] = $value;
+    }
+}
+
+class jTpl
```

Comment:
You cannot redefine classes that already exists, like jTpl or lizmapTiler, else we couldn't test components which are using them and need real results from them.And we couldn't test these classes too (at least lizmapTiler).